ios/FluentUI/DrawerVnext/Drawer.swift

```diff
@@ -157,20 +207,36 @@ public struct Drawer<Content: View>: View {
     private func dragGesture(screenWidth: CGFloat) -> some Gesture {
         DragGesture()
             .onChanged { value in
-                let withinDragBounds = state.presentationDirection == .left ? value.translation.width < 0 : value.translation.width > 0
-                if withinDragBounds {
-                    horizontalDragOffset = value.translation.width
-                }
+                let velocity = value.translation.width
```

Comment:
I found this bug on the video below.
If you add logic like the snippet below to skip the updateTransition it'll fix it:

```
                if (slideOutDirection == .right && value.startLocation.x > value.location.x) ||
                    (slideOutDirection == .left && value.startLocation.x < value.location.x) {
                    return
                }
```


https://user-images.githubusercontent.com/68076145/104497752-add47b80-558f-11eb-9e09-bc7b4f1ffe2a.mov

Reply:
thanks for catching this, I have update the code with the fix